core/tauri/src/app.rs

```diff
@@ -204,6 +205,21 @@ pub struct AppHandle<R: Runtime> {
   tray_handle: Option<tray::SystemTrayHandle<R>>,
 }
 
+impl<R: Runtime> AppHandle<R> {
+  /// Initializes a webview window builder with the given window label and URL to load on the webview.
+  ///
+  /// Data URLs are only supported with the `window-data-url` feature flag.
+  pub fn window_builder<L: Into<String>>(&self, label: L, url: WindowUrl) -> WindowBuilder<R> {
```

Comment:
I know we use this exact pattern in other place too, but can we "invert" this interface?
I.e. instead of a method on `AppHandle` we pass `AppHandle` to the `WindowBuilder::new` constructor?
This "inline constructor" feels a bit off to me 🤔

Reply:
I agree, this is how it is in tao too.

Reply:
yeah I like that idea, i'll change that

Reply:
Maybe we can review rest of Tauri and make everything consistent?
Sure that would be a breaking change but a good one maybe 🤔

Reply:
Well that depends on the size of the refactor/breakingchange and the impact on post v1 work.